ckan/lib/navl/validators.py

```diff
@@ -32,7 +32,8 @@ def not_missing(key, data, errors, context):
 def not_empty(key, data, errors, context):
 
     value = data.get(key)
-    if not value or value is missing:
+    valid_values = [False, 0, 0.0]
```

Comment:
0 and 0.0 are the same for `in` test below

```suggestion
    valid_values = [False, 0]
```